README.md

```diff
@@ -46,6 +46,14 @@ For å installere node trenger man både *nodejs-legacy* og *npm*:
 sudo apt-get install nodejs-legacy npm git
 ```
 
+#### Mac-brukere merk dette! 
+For å kjøre gulp trenger du *pandoc*. Den enkleste måten å installere pandoc er
```

Comment:
Hei,

Her har du feilmeldinga som ramlet ut: 

```
$ ./gulp 
warn:    --minUptime not set. Defaulting to: 1000ms
warn:    --spinSleepTime not set. Your script will exit if it does not stay up for at least 1000ms
[13:01:14] Working directory changed to ~/Dropbox/repos/mine/oppgaver/codeclub_lesson_builder
[13:01:17] Using gulpfile ~/Dropbox/repos/mine/oppgaver/codeclub_lesson_builder/gulpfile.js
[13:01:17] Starting 'build'...
[13:01:17] Starting 'build-indexes'...
[13:01:17] Starting 'css'...
[13:01:17] Starting 'browserify'...
[13:01:17] Starting 'assets'...
⧗ Downloading Pandoc (~20-50MB depending on OS). This may take a minute or so.
metalsmith-pandoc: trouble with pandoc-bin installation
metalsmith-pandoc: trying to use system installed pandoc
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: spawn /Users/bjorn/Dropbox/repos/mine/oppgaver/codeclub_lesson_builder/node_modules/metalsmith-pandoc/node_modules/pandoc-bin/vendor/pandoc ENOENT
    at exports._errnoException (util.js:874:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:439:9)
    at process._tickCallback (node.js:353:17)
error: Forever detected script exited with code: 1
error: Script restart attempt #1
```

Forsvant da jeg installerte pandoc fra brew.